src/main/java/cz/mzk/utils/FileWriterUtils.java

```diff
@@ -0,0 +1,45 @@
+package cz.mzk.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.*;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Slf4j
+public class FileWriterUtils {
```

Comment:
Utils je spis postfix pro statickou tridu, ktera ma sadu utilit
tady mas wrapper pro BufferedWriter, Path a proste wrapper pro soubor, do ktereho zapisujes
takze je to semanticky matouci kdyz davas wrapperu postfix utils
dokonce nemusi to byt i Writter, budes pozdeji pouzivat wrapper i na cteni ze souboru

co treba FileWrapper?

Reply:
Ked si to teraz citam, tak to dava celkom zmysel :) okej, FileWrapper znie ovela lepsie